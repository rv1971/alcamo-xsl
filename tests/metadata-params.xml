<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc
    xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#"
    xmlns:dc="http://purl.org/dc/terms/"
    xmlns:owl="http://www.w3.org/2002/07/owl#"
    dc:identifier="metadata-params"
    dc:title="Test for metadata parameters"
    dc:creator="Alice"
    dc:publisher="example.org"
    dc:created="2023-04-18T11:28:00+02:00"
    dc:modified="2023-04-18T11:29:23+02:00"
    dc:accessRights="public"
    dc:rights="© Example.org 2023"
    dc:rigtsHolder="Example.org"
    owl:versionInfo="0.42.0">
  <a:section>
    <dc:identifier>metadata-params section</dc:identifier>
    <dc:title>Test for metadata parameters section</dc:title>
    <dc:creator>Bob</dc:creator>
    <dc:publisher>example.info</dc:publisher>
    <dc:created>2023-04-18T11:35:00+02:00</dc:created>
    <dc:modified>2023-04-18T11:35:44+02:00</dc:modified>
    <dc:accessRights>restricted</dc:accessRights>
    <dc:rights>© Example.info 2023</dc:rights>
    <dc:rigtsHolder>Example.info</dc:rigtsHolder>
    <owl:versionInfo>0.42.43</owl:versionInfo>>
  </a:section>

  <xsl:stylesheet
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:owl="http://www.w3.org/2002/07/owl#"
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0">
    <xsl:import href="../xsl/metadata.xsl"/>

    <xsl:template match="/">
      <a:result>
        <a:item>
          <xsl:value-of select="'dc:identifier '"/>
          <xsl:value-of select="$dc:identifier"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:title '"/>
          <xsl:value-of select="$dc:title"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:creator '"/>
          <xsl:value-of select="$dc:creator"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:publisher '"/>
          <xsl:value-of select="$dc:publisher"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:created '"/>
          <xsl:value-of select="$dc:created"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:modified '"/>
          <xsl:value-of select="$dc:modified"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:accessRights '"/>
          <xsl:value-of select="$dc:accessRights"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:rights '"/>
          <xsl:value-of select="$dc:rights"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'dc:rightsHolder '"/>
          <xsl:value-of select="$dc:rightsHolder"/>
        </a:item>

        <a:item>
          <xsl:value-of select="'owl:versionInfo '"/>
          <xsl:value-of select="$owl:versionInfo"/>
        </a:item>
     </a:result>
    </xsl:template>
  </xsl:stylesheet>
</a:doc>
