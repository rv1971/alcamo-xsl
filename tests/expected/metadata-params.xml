<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#" xmlns:dc="http://purl.org/dc/terms/" xmlns:owl="http://www.w3.org/2002/07/owl#">
  <a:item>dc:identifier metadata-params</a:item>
  <a:item>dc:title Test for metadata parameters</a:item>
  <a:item>dc:creator Alice</a:item>
  <a:item>dc:publisher example.org</a:item>
  <a:item>dc:created 2023-04-18T11:28:00+02:00</a:item>
  <a:item>dc:modified 2023-04-18T11:29:23+02:00</a:item>
  <a:item>dc:accessRights public</a:item>
  <a:item>dc:rights &#xA9; Example.org 2023</a:item>
  <a:item>dc:rightsHolder </a:item>
  <a:item>owl:versionInfo 0.42.0</a:item>
</a:result>
